server/classes/sensorContact.js

```diff
@@ -80,17 +80,18 @@ export default class SensorContact {
     const maxDistance = this.type === "planet" ? 1 + this.size / 2 : 1.1;
     if (yaw) {
       // Rotate the contact about the center
-      const destinationPoints = calculateRotatedPoint(this.destination, yaw);
```

Comment:
I can see the value in this, but I think it might be confusing to people. They update their Thorium and suddenly rotation nudges go backwards? That sounds like a breaking change to me.

Could you put it behind a toggle, maybe called "Invert Rotation Nudge", with the default set to the current behavior? When it's toggled on, it can do this inverted nudging.
<img width="192" alt="Screenshot 2023-04-18 at 9 21 16 AM" src="https://user-images.githubusercontent.com/6558157/232790502-699f6d7f-24d5-4237-9cfd-45c3fa62cc86.png">

Reply:
Really this should only affect the call from thrusters to nudge contacts. I was not aware of the sensors nudge that also called nudge.
I'll get a commit prepped to just affect auto-thrusters.

Reply:
Yeah, those rotate-y icons also call nudge.TBH, I forgot that auto-thrusters calls nudge too 😅